src/sheldon_sup.erl

```diff
@@ -0,0 +1,64 @@
+%%% @hidden
+%%% @doc Main Sheldon's supervisor.
+%%%
+%%% Copyright 2016 Inaka &lt;hello@inaka.net&gt;
+%%%
+%%% Licensed under the Apache License, Version 2.0 (the "License");
+%%% you may not use this file except in compliance with the License.
+%%% You may obtain a copy of the License at
+%%%
+%%%     http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%% Unless required by applicable law or agreed to in writing, software
+%%% distributed under the License is distributed on an "AS IS" BASIS,
+%%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%% See the License for the specific language governing permissions and
+%%% limitations under the License.
+%%% @end
+%%% @copyright Inaka <hello@inaka.net>
+%%%
+-module(sheldon_sup).
+-author("Felipe Ripoll <ferigis@gmail.com>").
+-behaviour(supervisor).
+
+%% API
+-export([start_link/0]).
+
+%% Supervisor callbacks
+-export([init/1]).
+
+-define(SERVER, ?MODULE).
```

Comment:
At Inaka, we don't like macros, so please use `?MODULE` everywhere instead of `?SERVER` or create a function to get this value.